### DISCUSSION TITLE
Run outside of the Desktop Environment in 240p?

### DISCUSSION THREAD

**Discussion by vol-2** [pystardust/ytfzf#703] [Q&A]
I am looking for a Youtube downloader program I can use to watch videos on my setup that uses an old RGB CRT arcade monitor. In the past I was able to do this with omx player, but it stopped working in like 2015-16 when youtube changed the way they did things and support was broken for 240p output via the GPIO. It had something to do with the overlays and FKMS and KMS when they transitioned from Pi3 to Pi4 stuff

Can anyone tell me if ytfzf can be run from the command line without being inside of XORG or some other desktop environment first? This is important to what I'm doing because the whole rest of the setup is command line only, and the monitor is RGB and 240p/480i

Also, I don't know if this is relevant, but I'd only be able to play 240p quality videos; back when I was using omxplayer for this, I could easily filter the video results to only show the 240p video quality for everything, so I would never accidentally try to play a video with a higher resolution. It didn't mess anything up if I tried, but it just wouldn't play anything at all (showed a blank screen)

Thanks

**Comment by Euro20179** [comment]
**This will probably only work with the linux kernel**

If you have mpv compiled with the drm video output driver then this should work (I believe it's usually compiled in by default)

As for thumbnails, I'd recommend `catimg` or `chafa` unless you have 2 monitors, then you could try `mpv` for the thumbnail viewer but it probably wont work as desired.

Ofc you could go without thumbnails, if you want to do that, dont use `-T chafa` (as shown later)

> Also, I don't know if this is relevant, but I'd only be able to play 240p quality videos

As for this, i'd recommend using the `-f`  option to choose the video quality.

In full the command would look something like this

`ytfzf -u mpv --url-handler-opts='--vo=drm' -T chafa -f search query`

`mpv` is the default player so `-u mpv` is not required.

I have no idea if this will work, i just know that this works on my computer outside of xorg/wayland

**Reply by vol-2** [reply]
Ok. Thanks for the reply. I guess I'm a pioneer with this. I am definitely still able to get 240p out of the GPIO, but only with strictly command line programs. If it runs outside of xorg/wayland, then there is enough hope that I will try it and see if it works.
